tests/services/cli_test.py

```diff
@@ -191,36 +197,39 @@ def tearDown(self):
 
     def test_cli(self, test_cases):
         """Test if the CLI hits specific client methods."""
-        for commandline, client_func, func_args,\
-            func_kwargs, config_string, config_expect\
-                in test_cases:
-            try:
-                args = shlex.split(commandline)
-                env_config = cli.DefaultConfig(
-                    json.load(StringIO.StringIO(config_string)))
-
-                config = cli.main(
-                    args,
-                    env_config,
-                    CLIENT,
-                    parser_cls=MockArgumentParser)
-
-                if client_func is not None:
-                    client_func.assert_called_with(*func_args, **func_kwargs)
-
-                # Check attribute values
-                for attribute, value in config_expect.iteritems():
-                    self.assertEqual(
-                        getattr(config, attribute),
-                        value,
-                        'Attribute value unexpected: {}, {}'.format(
-                            attribute,
-                            value))
-
-            except ArgumentParserError as e:
-                self.fail('Argument parser failed on {}, {}'.format(
-                    commandline,
-                    e.message))
+        tmp_config = os.path.join(self.test_dir, '.forseti')
+        with mock.patch.dict(
+            os.environ, {'FORSETI_CLIENT_CONFIG': tmp_config}) as mock_config:
```

Comment:
nit: you don't need to use 'as var' if you don't need to access the var in the context.